web/packages/extension/src/ruffle.ts

```diff
@@ -1,9 +1,20 @@
+/**
+ *
+ * This script runs in the MAIN ExecutionWorld for the following reasons:
+ *
+ * 1. On Chrome, you are explicitly banned from registering custom elements.
+ * 2. On Firefox, you can register custom elements but they can't expose any
+ *    useful API surface, and can't even see their own methods.
+ *
+ */
+
 import { Setup } from "ruffle-core";
 import { Message } from "./messages";
 
 function handleMessage(message: Message) {
     switch (message.type) {
         case "load": {
+            __webpack_public_path__ = message.publicPath;
```

Comment:
Change the webkit public path to the full URL of the extension's /dist/ folder.

Reply:
Could we check that this is using an extension protocol?

Reply:
If I do the `new URL` here, yes. https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getURL#return_value is a string, and it's hard to check the protocol of a string.

Reply:
Resolved in #20264